### ISSUE TITLE
java.lang.IncompatibleClassChangeError: Method 'org.bukkit.attribute.Attribute[] org.bukkit.attribute.Attribute.values()' must be Methodref constant

### ISSUE THREAD

**Issue by mk7luke** [PlayPro/CoreProtect#685]
purpur 1.21, CoreProtect v23.2-RC1 (am I just using an incorrect plugin version?). This occurs on any entity death:

```[Server thread/ERROR]: Could not pass event EntityDeathEvent to CoreProtect v23.2-RC1
java.lang.IncompatibleClassChangeError: Method 'org.bukkit.attribute.Attribute[] org.bukkit.attribute.Attribute.values()' must be Methodref constant
	at CoreProtect-Edge-23.2-RC1.jar/net.coreprotect.listener.entity.EntityDeathListener.logEntityDeath(EntityDeathListener.java:287) ~[CoreProtect-Edge-23.2-RC1.jar:?]
	at CoreProtect-Edge-23.2-RC1.jar/net.coreprotect.listener.entity.EntityDeathListener.onEntityDeath(EntityDeathListener.java:554) ~[CoreProtect-Edge-23.2-RC1.jar:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor241.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-mojangapi-1.21-R0.1-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:77) ~[paper-mojangapi-1.21-R0.1-SNAPSHOT.jar:1.21-2258-ff19bcc]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-mojangapi-1.21-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-mojangapi-1.21-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:998) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.world.entity.LivingEntity.dropAllDeathLoot(LivingEntity.java:1929) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.world.entity.LivingEntity.die(LivingEntity.java:1830) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.world.entity.LivingEntity.hurt(LivingEntity.java:1635) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.world.entity.monster.Zombie.hurt(Zombie.java:358) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.world.entity.player.Player.attack(Player.java:1383) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.level.ServerPlayer.attack(ServerPlayer.java:2542) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.network.ServerGamePacketListenerImpl$4.onAttack(ServerGamePacketListenerImpl.java:2878) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket$1.dispatch(ServerboundInteractPacket.java:29) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.dispatch(ServerboundInteractPacket.java:89) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleInteract(ServerGamePacketListenerImpl.java:2787) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:76) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:14) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:36) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:151) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1576) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:201) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:125) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1553) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1546) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:135) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1505) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1512) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1357) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:332) ~[purpur-1.21.jar:1.21-2258-ff19bcc]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]```

**Comment by Intelli** [comment]
Should be resolved in the latest release (Edge 23.2 RC3)

**Comment by mk7luke** [comment]
Thank you, yes it is, that was incredible timing lol

> Should be resolved in the latest release (Edge 23.2 RC3)
